products/llbuild-analyze/Sources/json.swift

```diff
@@ -0,0 +1,151 @@
+// This source file is part of the Swift.org open source project
+//
+// Copyright 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+
+
+import llbuildAnalysis
+import llbuildSwift
+
+// MARK: - JSON Serialization
```

Comment:
`json.swift` is a bad name for this file, IMHO

Reply:
Would `JSONSerialization` be better?

Reply:
I think `PathOutput` would be better, since that is the key top it defines, I think?

Reply:
`CriticalBuildPathSerialization`? :D

Reply:
It just contains the serialization in json format though. We support a different one as well: GraphViz. And we might add more in the future (there's an open task for Chromium tracing for example).

Reply:
I've renamed it to `PathJSONSerialization` and the GraphViz.swift file to `PathGraphVizSerialization`. I think using `Path` instead of `CriticalPath` in this context is fine since the whole tool is built for finding the critical path - although the serialization is generic and can visualize any path theoretically.

Reply:
👍